src/script/components/list/ConversationListCell.tsx

```diff
@@ -55,6 +55,7 @@ export interface ConversationListCellProps {
   showJoinButton: boolean;
   handleArrowKeyDown: (e: React.KeyboardEvent) => void;
   isFocused?: boolean;
+  setDefaultFocus: () => void;
```

Comment:
can you document what this function is supposed to do. It's not quite clear for the code itself